lang/python/python-curl/Makefile

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright (C) 2010 OpenWrt.org
```

Comment:
Don't assign copyright to OpenWrt unless you're affiliated, use your own or omit it.